docs/bcfusion/users/gov.md

```diff
@@ -0,0 +1,122 @@
+
+BSC daos are created on Tally both for the mainnet and testnet.
+
+* Testnet: https://www.tally.xyz/gov/bnb-chain-testnet
+* Mainnet: to be announced
+
+## Parameters
+
+There are several parameters which will affect the governance process on the BSC.
+Especilly, the governance process on the BSC only enabled after enough voting power is migrated from the Beacon Chain
+to the BSC (i.e., the `startGovThreshold` parameter).
+
+| Parameter               | Description                                                                      | Mainnet Value | Testnet Value |
+|-------------------------|----------------------------------------------------------------------------------|---------------|---------------|
+| votingDelay             | a fixed duration  after which users can vote to a proposal                       | 0 hour        | 0 hour        |
+| votingPeriod            | the voting period before tally                                                   | 7 days        | 1 day         |
+| proposalThreshold       | a fixed amount of gov BNB needed for a proposal                                  | 200 govBNB    | 100 govBNB    |
+| quorumNumberRator       | the percentage of the total voting power required to produce a final vote result | 10%           | 10%           |
+| startGovThreshold       | the total supply of gov token to enable the gov function                         | 10M BNB       | 10M BNB       |
+| minPeriodAfterTheQuorum | the time to add for voting when a proposal reaches quorum                        | 1 day         | 1 hour        |
+| timerlockDelay          | the timer locker duration to execute a proposal                                  | 1 day         | 6 hours       |
+
+## Governance Process Guide
+
+You need to connect to your Web3 wallet (e.g., TrustWallet, BEW, Metamask) for the following operations.
+
+### Delegate Voting Power
+
+After you have delegated your BNB to a BSC validator, you can start participating in the BSC governance.
+To participate in BSC governance, you first need to delegate your voting power to a validator or yourself if you wish to
+vote directly.
+
+<img src={Tally1} width="800"/>
+
+You can click the `My voting power` button on the top right corner of the screen to delegate your voting power.
+
+<img src={Tally2} width="800"/>
+
+You can delegate your voting power to yourself if you want to vote/create proposals directly, or to others if you want
+him/her to vote/create proposals on your behalf.
+
+<img src={Tally3} width="800"/>
+
+If you delegate the voting power to yourself, you will see the current number of your voting power to participate in the
+governance.
+
+### Create Proposals
+
+If you have sufficient voting power (i.e., greater than the `proposalThreshold`), you can create proposals on the BSC
+network.
+
+To create a proposal, click on the "Create new proposal" button on the top right corner of the screen.
```

Comment:
please hightlight to user that: one user can have only one active on pending proposal at the same time to prevent abuse.

Reply:
added